src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -107,21 +108,23 @@ public void runAppCommand(List<String> args) throws ProcessRunnerException {
     command.addAll(args);
 
     command.add("--quiet");
-    command.addAll(GcloudArgs.get("credential-file-override", appCommandCredentialFile));
+
+    Map<String, String> environment = Maps.newHashMap();
+
+    if (appCommandCredentialFile != null) {
```

Comment:
Good point.  I was under the misapprehension that outputCommand did
something important. I did some cleanup to make that clearer.

On Thu, Jun 9, 2016 at 11:15 AM, Etan Shaul <notifications@github.com>
wrote:

> In src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/113#discussion_r66460675>
> :
>
> > @@ -107,21 +108,23 @@ public void runAppCommand(List<String> args) throws ProcessRunnerException {
> >      command.addAll(args);
> >
> >      command.add("--quiet");
> > -    command.addAll(GcloudArgs.get("credential-file-override", appCommandCredentialFile));
> > +
> > +    Map<String, String> environment = Maps.newHashMap();
> > +
> > +    if (appCommandCredentialFile != null) {
>
> Minor, but the flow of this might be better if it was moved next to the
> other environment setting logic (without format in between).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/113/files/eefe29a60f4531250f6c6a01f9b42ec2679dc357#r66460675>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HQOWyzYtH0iH-4Gft_XOtaesxnuuks5qKC4EgaJpZM4IyE-d>
> .
>